tests/storage_tests.cpp

```diff
@@ -259,7 +269,7 @@ TEST_CASE("issue880") {
                                 foreign_key(&X::fkey_Rendimiento).references(&Rendimiento::id)));
     storage.sync_schema();
 
-    Fondo fondo{5};
```

Comment:
why did you remove 5?

Reply:
What's the point?

Reply:
5 is checked at runtime. If you remove 5 it will be equal to 0. I know that the result is not checked and it will not alter UT results. But I don't see the reason to change this. Anyway it is just a question

Reply:
The reason was simply not having to provide a conditional constructor (due to a compiler's possibly missing default member initializer support).
But you have a valid point. Let me change it back.